src/docs/development/ui/assets-and-images.md

```diff
@@ -280,7 +280,7 @@ The `lib/` is implied, so it should not be included in the asset path.
 Flutter assets are readily available to platform code via
 AssetManager on Android and NSBundle on iOS.
 
-### Android
+### Loading Flutter Assets in Android
```

Comment:
Can you use sentence case? In this case, it only affects one word.
=> Loading Flutter assets in Android

Reply:
done